source/client-side-encryption/tests/README.rst

```diff
@@ -2509,6 +2509,17 @@ options::
       },
```

Comment:
should the `clientEncryptionOptions` also include the `aws` kms provider?

Reply:
Yes. Good eye. Updated.